cli/crates/common/src/analytics.rs

```diff
@@ -4,13 +4,13 @@ use crate::{environment::Environment, errors::CommonError};
 use chrono::{DateTime, Utc};
 use core::panic;
 use derivative::Derivative;
-use once_cell::sync::OnceCell;
```

Comment:
Can we take the crate out from `Cargo.toml`?

Reply:
Yeah